src/Page/Object.elm

```diff
@@ -533,6 +581,20 @@ view session model =
                         , toCategory = \_ -> ""
                         }
 
+                AddTransformModal component index autocompleteState ->
+                    AutocompleteSelectorView.view
+                        { autocompleteState = autocompleteState
+                        , closeModal = SetModal NoModal
+                        , footer = []
+                        , noOp = NoOp
+                        , onAutocomplete = OnAutocompleteAddTransform component index
+                        , onAutocompleteSelect = OnAutocompleteSelectTransform component index
+                        , placeholderText = "tapez ici le nom d'un procédé de transformation pour le rechercher"
+                        , title = "Sélectionnez un procédé de transformation"
+                        , toLabel = .name
```

Comment:
Strange I thought that I did a review comment here… Shouldn't we use the `displayName` here and fallback to `name`?